tests/postgres_tests/test_ranges.py

```diff
@@ -405,22 +405,82 @@ def test_serialize_range_with_null(self):
 
 class TestValidators(PostgreSQLTestCase):
 
-    def test_max(self):
+    def test_numericrange_max(self):
```

Comment:
While testing for None is certainly needed, I'm not sure testing for different types of ranges adds value. We are using the same class of validator, so testing the different range types tests, essentially, that `<` and `>` work for dates as well as they do for numbers (not to mention that other types of ranges are possible).